lib/lbt/bundle/AutoSplitter.js

```diff
@@ -226,7 +228,20 @@ class AutoSplitter {
 				props.store(out, "");
 				return out.toString().length();
 				*/
-				return fileContent.toString("latin1").length;
+
+				// input encoding is read from project configuration, default is ISO-8859-1
+				const encoding = resource.getProject() && resource.getProject().resources.propertiesFileEncoding;
```

Comment:
I'm wondering, why is this necessary?

If the `escapeNonAsciiCharacters` tasks was executed, all .properties resources should be UTF-8 compatible by now.

Or are these resources coming from dependencies and not the project itself?

Reply:
the latter